app/src/main/res/layout/focus_preference.xml

```diff
@@ -60,7 +60,6 @@
             android:layout_below="@android:id/title"
             android:layout_alignStart="@android:id/title"
             android:textAppearance="?android:attr/textAppearanceListItemSecondary"
-            android:textColor="?android:attr/textColorSecondary"
```

Comment:
Was this in error?

Reply:
Is redundant, but I will revert this change because is somehow out of scope